src/scrape_up/github/users.py

```diff
@@ -118,6 +118,30 @@ def repo_count(self):
         except:
             message = f"No. of Repos not found for username {self.username}"
             return message
+    def get_Profile_Views_Count(self):
+
+        """
+        Fetch the Profile Views count of Github user profile.
+        """
+        mainPageData = self.__scrape_page()
+        try:
+            hyperLinks = mainPageData.find_all('a')
+            for a_tag in hyperLinks:
+                img_tag = a_tag.find('img')
+                if img_tag:
+                    if f'data-canonical-src="https://komarev.com/ghpvc/?username=' in str(img_tag):
+                        url =img_tag['data-canonical-src']
+                        response = requests.get(url)
+                        html_content = response.text
+                        soup = BeautifulSoup(html_content, 'html.parser')
+                        value_element = soup.find_all('g')
+                        for text_tag in value_element:
+                            if f'fill="#fff"' in str(text_tag): 
+                                valueLst=text_tag.find_all('text')
+                                return valueLst[-1].text
+        except:
```

Comment:
Needs few more changes @yyuvraj54 
+ Return the output with a data and message, you can check other methods for reference. 
+ Mention this method in the `documentation.md` file also.
+ Upload SS of the output for the validation.

Reply:
ok working on it

Reply:
When profile views are available
![corrr](https://github.com/Clueless-Community/scrape-up/assets/30363687/764ae727-dae1-4b0c-bf10-8d7de833587f)


When profile views are not available
![not](https://github.com/Clueless-Community/scrape-up/assets/30363687/2f1d10a8-7085-47e5-9f03-09428430e888)